health-checks.js

```diff
@@ -8,7 +8,7 @@ const newman = require('newman');
  * We need to make sure that the program has read/write access to the file system and that newman has been properly installed.
  */
 function runHealthChecks(tempFolder, callback) {
-  const folderPath = path.join(__dirname, tempFolder);
+  const folderPath = path.join('.', tempFolder);
```

Comment:
What happens if the tempFolder provided in the cli is an absolute path?

Reply:
I fixed this, the reportFolder path is now converted to an absolute when relative, so that the health checks don't have to take care of it